pkg/hubble/relay/pool/manager.go

```diff
@@ -186,6 +194,38 @@ func (m *PeerManager) manageConnections() {
 	}
 }
 
+func (m *PeerManager) reportConnectionStatus() {
+	connTimer, connTimerDone := inctimer.New()
+	defer connTimerDone()
+	for {
+		select {
+		case <-m.stop:
+			return
+		case <-connTimer.After(m.opts.connStatusInterval):
+			m.mu.RLock()
+			connStates := make(map[connectivity.State]uint32)
+			var nilConnPeersNum uint32 = 0
+			for _, p := range m.peers {
+				p.mu.Lock()
+				if p.conn == nil {
+					nilConnPeersNum++
+					p.mu.Unlock()
+					continue
+				}
+				state := p.conn.GetState()
+				// if does not exists, num is 0 (zero value)
+				// which happens to be a perfect initial value
+				num, _ := connStates[state]
+				num++
+				connStates[state] = num
```

Comment:
nit: IMO it's fine to "silently" rely on zero values:
```suggestion
				connStates[state] = connStates[state] + 1
```

Reply:
Sure, applied your suggestion.